### ISSUE TITLE
Total Power wasted by this plugin

### ISSUE THREAD

**Issue by gutjuri** [vtoubiana/TrackMeNot#137]
A query on a search engine such as Google requires a non-trivial amount of power. By doing a large amount of (useless) queries, this plugin wastes power and contributes to climate change.

I am aware that privacy is a valuable thing, but this plugin's means of ensuring it introduce a tradeoff between power consumption and privacy. This should be made clear in the UI. 

I ask you to display to the user an estimate of the power he/she wasted by having this plugin installed. Ideally, this would be in the unit of kWh, as well as a more intuitive unit (e.g. litres of water that could have been heated to boiling point with this energy). Additionally, i'd be great to display the number of kg of CO2 emmitted as a result of this plugin.

Please don't see this comment as trolling. I am only concerned that this plugin's potential consequenses are transparent to the user.

**Comment by hack-r** [comment]
You're wrong about the non-trivial part... and perhaps Google could better reduce its environmental impact by discontinuing surveilling users, selling their data, and giving it away to all interested parties in corporate Town Halls (?)...

**Comment by gutjuri** [comment]
> You're wrong about the non-trivial part... 

Citation needed, last time I checked a google search was around 0.3Wh (https://fullfact.org/environment/google-search/). Boiling one litre of water takes roughly 93Wh (https://www.quora.com/How-much-energy-is-needed-to-boil-water-How-much-energy-would-it-take-to-boil-1-liter-of-water?share=1), equivalent to 310 useless google searches. If you leave trackmenot on default settings, it performs 10 queries per minute. That means that every 31 minutes of using trackmenot you boil a litre of water and throw it away.

> and perhaps Google could better reduce its environmental impact by discontinuing surveilling users, selling their data, and giving it away to all interested parties in corporate Town Halls (?)...

Agreed, but good luck convincing google of that. The path forward into a privacy-friendly world not destroyed by climate change should include users making informed decisions on the actions they take that impact privacy and energy consumption, which is what I proposed.

**Comment by darmanilink** [comment]
How many gallons of heated water do you use every time you do the dishes? Do laundry? Take a shower?
How much energy does playing a AAA game from the last year with your fans blazing as to not damage your GPU take?
How much energy does binging a show on netflix, amazon or any other streaming service take?
The amounts of energy you're talking about is extremely trivial and frankly the amount this "contributes" is negligible on the scale of cities. Running a street lamp for a night takes more energy.

Google tracks you even if you have cookies disabled and don't use their search engine. Every time you visit a website with any sort of analytics google gets that info. 
Denying them info ensures that they get less, but more accurate information about you.
Flooding them with false positives gives them a ton of info and junks your entire profile. Its why this extension exists.

**Comment by hack-r** [comment]
Maybe we can cut the powerlines to their server farms?

I ain't doing no dishes man. Door dash.

**Comment by dhowe** [comment]
It's always interesting how people's normative judgments of value implicitly enter into considerations of resource use. 

Using truly massive resources for Netflix/Porn/Zoom, for Machine Learning, for Cryptocurrencies, are all perfectly Ok! 
But privacy... well, that's a different story

**Comment by furu00** [comment]
whataboutism 🙄
last time he checked.. "the headline"
his own link says:
`Nevertheless, even using Google’s 2009 estimate of a search requiring 0.3 Wh, Mr Liddle’s estimate is far too high. The energy required to power a Google search could power a 10 watt bulb for around 108 seconds, not an hour. That’s 33 times less energy than Mr Liddle’s estimate.`
🤣

**Comment by Thomashighbaugh** [comment]
> This is the same hysteria form a half-understanding that created the dumb laws in california on gaming computers.

This statement should be amended to the following to be most accurate: 

> This is the same hysteria form a half-understanding that created the dumb laws in california.

Because almost none of the laws passed in California are not toothless appeals to the hysterics of the voting public that if violated are inconsequential or if some consequence does exist it does not actually profit the people in that cesspool called Sacramento pushing the bill financially while pricing out smaller firms from entering what market that the law is intended to regulate. 

- A salty 6th generation Bay Area native
